how-it-works/3_governance/01_network-nervous-system-nns.subpage.md

```diff
@@ -5,27 +5,91 @@ shareImage: /img/how-it-works/network-nervous-system.webp
+
+
+## Proposals
+_Proposals_ are suggestions on how to change the ICP protocol.
+Any neuron can submit a proposal, and all other neurons can then vote on it.
+Neurons can vote with the voting power they have at the time of the proposal's submission.
+
+
+## Liquid democracy
+The NNS implements a form of _liquid democracy_, which means that neurons can either vote directly on proposals, or delegate their voting power to other neurons, called _following_.
+Following is done on the basis of proposal _topics_, i.e. groups of proposals that are similar.
+This facilitates following different experts on different topics.
+The voting delegations can be changed at any point in time.
+
+
+## Voting rules
+Voting on NNS proposals follows an algorithm called _wait-for-quiet_.
+The main idea of wait-for-quiet is that proposals where all voters agree can be executed quickly, while strongly contested proposals have a longer voting period. This allows voters to have enough time to react if a proposal result is turned all of a sudden.
+Currently, most proposals start with an initial voting period of 4 days that can be extended up to a total of 8 days.
+A proposal is adopted if either of the two conditions are met:
+1) the voting period has elapsed, there are more votes in favor than against the proposal (simple majority), and at least 3% of the totally available voting power voted in favor of the proposal (enough participation).
+2) more than 50% of the total voting power are in favor of the proposal. At this point, the proposal's result cannot be changed anymore even if all remaining voters vote against the proposal and therefore the proposal is executed without awaiting the end of the voting period.
+
+
+## Proposal execution
+As soon as a proposal is adopted, it is executed by the NNS. With very few exceptions, proposals are executed automatically and fully on-chain.
+
+
+
+
+## What the NNS governs
+The NNS governs the full Internet Computer blockchain.
+For example, the NNS makes decisions on the following topics.
+* *Network topology*: The NNS decides which node machines can join the network and how these nodes are organized into _subnets_.
+* *Software run on the nodes*: The NNS decides which software is run by the nodes that form the blockchain. This allows the NNS to upgrade the protocol without manual intervention of the individual node operators.
+* *Tokenomics*: The NNS decides how many rewards node operators and governance participants get and the costs of computation and storage on the Internet Computer.
+* *Governance*: The NNS decides on the rules of the governance itself, e.g. how many votes are needed for a proposal to be adopted, how long the voting period is, etc.
+
+
+All these decisions are executed fully on-chain and at low cost compared to other chains. Therefore, decisions can be made efficiently and frequently allowing the NNS to adapt the ICP platform to the community's needs.
```

Comment:
A few lines above we state " With very few exceptions, proposals are executed automatically and fully on-chain." which seems in contrast to this sentence. Maybe we can write something like "With the exception of motion proposals ..."

Reply:
Added motion proposals as an example (to make sure it is clear what they are ) and then implemented you idea.